test/unit/comment_test.rb

```diff
@@ -308,4 +308,12 @@ class CommentTest < ActiveSupport::TestCase
     assert_not_nil graph
     assert graph.class, Hash
   end
+
+  test 'find email using twitter user name' do
+    require 'yaml'
+    config = YAML.load(File.read('test/fixtures/user_tags.yml'))
+    username = config["twitter3"]["data"]["info"]["nickname"]
```

Comment:
Also why don't you just use the string here directly? I think it's unnecessary to load the file and look for keys. You're only testing `find_email` so the lines below should suffice. When you replace `username` with the value you're you're looking for.